cli/kubectl_kadalu/logs.py

```diff
@@ -68,15 +69,17 @@ def run(args):
             container = '-c' + args.container
 
         if not args.podname:
-            cmd = utils.kubectl_cmd(args) + ["get", "pods", "-nkadalu", "-oname"]
+            cmd = utils.kubectl_cmd(args) + [
+                "get", "pods", "-nkadalu", "-oname"
+            ]
             resp = utils.execute(cmd)
             # Remove empty lines(pod-names) from command response
-            pods = resp.stdout.strip().split()
+            pods = resp.stdout.split()
```

Comment:
no need to remove whitespace or new lines?

Reply:
Intentional, `split` (without any args) has ability to remove any type (\n or \t etc) of blank spaces

```
>>> line = "\n\na\n\nb c d   e   f\n\ng\n h\n\n"
>>> print(line)


a

b c d   e   f

g
 h


>>> line.split()
['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h']
>>> 
```

Reply:
nice. 👍